core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalWriter.java

```diff
@@ -84,7 +68,10 @@ public void write(JournalEntry entry) throws IOException, JournalClosedException
     }
     Preconditions.checkState(entry.getAllFields().size() <= 2,
         "Raft journal entries should never set multiple fields, but found %s", entry);
-    if (mCurrentJournalEntrySize.get() > mFlushBatchBytes) {
+    // journal entry size max is the hard limit set by underlying ratis
+    // we use a smaller value to guarantee we don't pass the hard limit
+    if (mCurrentJournalEntrySize.get()
+        > ServerConfiguration.getBytes(PropertyKey.MASTER_EMBEDDED_JOURNAL_ENTRY_SIZE_MAX) / 3) {
```

Comment:
Same here. You might be surprised that the `ServerConfiguration.getXXX` are not really getters.

Reply:
fixed